cek/core/models.py

```diff
@@ -231,28 +231,28 @@ def verify_application_id(self, application_id):
 
 
 class LaunchRequest(Request):
-    """Type represents a LaunchRequest from CEK."""
+    """ This request is used to declare that the user has requested to use a this extension. """
```

Comment:
I find this explanation a bit confusing. It explains from the point of view of the CEK server, but I think it would be easier to understand if we describe from the point of view of the user of this SDK and instead of referring to CEK we should refer to the user of the skill.

How about something like:
```Received when the user launch the skill.```

Reply:
Agree. Thanks for pointing out! :-)